handlers.js

```diff
@@ -255,7 +255,7 @@ module.exports = function(port) {
             identityCheckParams = assign({ip: (port.bus.config.debug && request.headers['x-forwarded-for'])
```

Comment:
I know this code is not part of the merge request but I just noticed the assign method above - the ip can be easily hacked in case a fake "ip" is sent as a property of the payload params object.